### DISCUSSION TITLE
Generate PDF in Region,Day order

### DISCUSSION THREAD

**Discussion by WoodieFL** [code4recovery/12-step-meeting-list#716] [Q&A] [answered]
The original half page PDF layout created a booklet in Region, Day order. However, we need a booklet with a page size of 5.5 x 8.5 and when we use the new Generate PDF, the only two sorting options are Day, Region or Day. It would be perfect for us if it had the Region, Day option as well. Is there any way to do this or could it be added to the Generate PDF page as an option?

I tried using Chris Toews' plugin, but it didn't give me the layout I need unfortunately.

**Comment by joshreisner** [comment, marked as answer]
I should be able to handle that! I moved [your issue](https://github.com/code4recovery/pdf/issues/24) to the PDF project repo
